docs/developers/tutorials/your-first-dapp.md

```diff
@@ -236,16 +236,16 @@ cd dapp
 
 Customize the Smart Contract Address
 
-Let's copy our `config.devnet.tsx` into a new file `config.tsx` and let's fill in the Smart Contract address constant with our smart contract address.
-
 ```sh
-cp src/config.devnet.tsx src/config.tsx
 nano src/config.tsx
 ```
 
-We'll take a look on the line 15:
+We'll take a look on the first instruction
```

Comment:
Please add `:` at the end of the sentence.

Reply:
done